python/setup.py

```diff
@@ -24,7 +24,7 @@
     "wheel",
 ]
 sklearn = ["scikit-learn"]
-torch = ["torch>=1.9.0", "torchvision>=0.11.0"]
+torch = ["torch>=1.8.0", "torchvision>=0.11.0"]
```

Comment:
do we need to change torchvision version as well?

Reply:
See https://docs.databricks.com/release-notes/runtime/8.3ml.html

Reply:
torchvision 0.9.1

Reply:
updated